setup.py

```diff
@@ -135,6 +135,7 @@ def run(self):
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: 3.13",
```

Comment:
Python 3.13 deprecated stdlib's [cgi](https://docs.python.org/3.11/library/cgi.html) module. [apistar](https://github.com/encode/apistar) uses it, but it ceased being maintained.
```python
import apistar
```
```python
ModuleNotFoundError: No module named 'cgi'
```

Reply:
apistar is exclusively used at this spot:

https://github.com/kennethreitz/responder/blob/f90955a9b9a201a5de21028795c32a8c0d2687df/responder/ext/schema/__init__.py#L128-L136

Do you see a way to replace that using a different library? Please advise.

/cc @taoufik07, @tomchristie, @tabotkevin